src/components/Common/Modal/ViewFollowModal/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from 'react';
+import { useRecoilValue } from 'recoil';
+
+import Avatar from '@/components/Common/Avatar';
+import { DEFAULT_PROFILE_IMAGE } from '@/constants/profile';
+import { userAtom } from '@/recoil/user';
+import { getUserDetailsById } from '@/services/Follow/follow';
+
+import {
+  FollowButton,
+  FollowButtonWrapper,
+  NoListMessage,
+  UserList,
+  UserListItem,
+  UserName,
+  ViewFollowWrapper,
+} from './style';
+
+export default function ViewFollowModal() {
+  const user = useRecoilValue(userAtom);
+  const [activeList, setActiveList] = useState('팔로잉');
+  const [userDetails, setUserDetails] = useState([]);
```

Comment:
```suggestion
const [userDetails, setUserDetails] = useState<User[]>([]);
```

userDetails의 타입이 지정되지 않아서 아래 detail을 참조하는 부분에서 빨간줄이 생기는 것 같아요!

`User[]`로 지정해주는게 필요해보입니다!

Reply:
앗 어쩐지 자꾸 빨간줄이 뜨더라구요 ㅎㅎ 저 부분을 놓쳤네용~ 타입 지정해두겠습니다 !!